Source/Forms/Gen8/WildPLA.cpp

```diff
@@ -0,0 +1,156 @@
+#include "WildPLA.hpp"
+#include "ui_WildPLA.h"
+
+#include <Core/Gen8/Generators/WildGeneratorPLA.hpp>
+#include <Core/Gen8/Profile8.hpp>
+#include <Core/Parents/ProfileLoader.hpp>
+#include <Forms/Controls/Controls.hpp>
+#include <Forms/Gen8/Profile/ProfileManager8.hpp>
+#include <Forms/Models/Gen8/RaidModel.hpp>
+#include <QMenu>
+#include <QSettings>
+
+WildPLA::WildPLA(QWidget *parent) : QWidget(parent), ui(new Ui::WildPLA)
+{
+    ui->setupUi(this);
+
+    setAttribute(Qt::WA_QuitOnClose, false);
+
+    updateProfiles();
+    setupModels();
+}
+
+WildPLA::~WildPLA()
+{
+    QSettings setting;
+    setting.beginGroup("WildPLA");
+    setting.setValue("profile", ui->comboBoxProfiles->currentIndex());
+    setting.setValue("geometry", this->saveGeometry());
+    setting.endGroup();
+
+    delete ui;
+}
+
+bool WildPLA::hasProfiles() const
+{
+    return !profiles.empty();
+}
```

Comment:
Is this function necessary?

Reply:
No, will remove. Extra from copying over other form as a template